### DISCUSSION TITLE
Drive details dashboard?

### DISCUSSION THREAD

**Discussion by bgraabek** [teslamate-org/teslamate#2772] [Q&A]
I started with version 1.26.1 (I think it was) almost a month ago and I upgraded to version 1.27 yesterday. From discussions and the change log I can see I should now be able to access tire pressure data in a “drive details” dashboard. I don’t see that anywhere?
Teslamate is running the standard docker containers.

**Comment by cwanja** [comment]
Should be below the elevation graphs in a specific drive detail query.

**Reply by bgraabek** [reply]
I think I expressed myself badly. It is the “drive details” dashboard that I'm not seeing, it is not on the list of available dashboards.
Maybe related to the issue of not seeing the “drive details” dashboard is the fact that when I switch dashboards, I now get the question if I wish to save the modifications to whatever dashboard I'm looking at, even though I haven't made any modifictions at all.

**Reply by cwanja** [reply]
Go to the Drives dashboard and then click or press the timestamp for a specific drive. That will open the Drive Details for a specific drive. And tire pressure will be listed there.

**Reply by bgraabek** [reply]
Thanks!
